cfme/services/catalogs/orchestration_template.py

```diff
@@ -1,166 +1,284 @@
 from cfme.fixtures import pytest_selenium as sel
-from cfme.web_ui import (
-    accordion, fill, form_buttons, menu, Form, Input, flash, AngularSelect, Select, ScriptBox)
 from cfme.web_ui import paginator as pg
-from cfme.web_ui import toolbar as tb
+from navmazing import NavigateToAttribute, NavigateToSibling
 from utils.update import Updateable
 from utils.pretty import Pretty
-from utils import error, version
-
-
-cfg_btn = partial(tb.select, "Configuration")
-accordion_tree = partial(accordion.tree, "Orchestration Templates")
-
-dialog_form = Form(
-    fields=[
-        ('dialog_name', Input("dialog_name")),
-        ('save_button', form_buttons.save)
-    ])
-
-create_template_form = Form(
-    fields=[
-        ('description', "textarea#description"),
-        ("template_type", {
-            version.LOWEST: Select("select#type"),
-            '5.5': AngularSelect('type')}),
-        ('content', ScriptBox(ta_locator="//div[@id='basic_info_div']/div/div"
-                    "/div/div/div/div/div/div/pre/span")),
-        ('template_name', Input("name")),
-        ('add_button', form_buttons.add),
-        ('edit_button', form_buttons.save),
-    ])
-
-
-def _orch_templates_create_dialog(context):
-    accordion_tree('All Orchestration Templates',
-        context['template_type'], context['template_name'])
-    cfg_btn('Create Service Dialog from Orchestration Template')
-
-
-menu.nav.add_branch(
-    'services_catalogs',
-    {
-        'orchestration_templates':
-        [
-            lambda _: accordion.click('Orchestration Templates'),
-            {
-                'select_template':
-                [
-                    lambda ctx: accordion_tree('All Orchestration Templates',
-                        ctx['template_type'], ctx['template_name']),
-                    {'edit_template': lambda _: cfg_btn("Edit this Orchestration Template"),
-                     'create_service_dialog': lambda _: cfg_btn("Create Service Dialog from "
-                        "Orchestration Template")}
-                ],
-                'orch_template_type':
-                [
-                    lambda ctx: accordion_tree(
-                        'All Orchestration Templates', ctx['template_type']),
-                    {'create_new_template': lambda _: cfg_btn("Create new Orchestration Template")}
-                ]
-            }
-        ]
-    }
-)
-
-
-class OrchestrationTemplate(Updateable, Pretty):
-    pretty_attrs = ['template_type', 'template_name']
-
-    def __init__(self, template_type=None, template_name=None, description=None):
+from utils.appliance import Navigatable
+from utils.appliance.implementations.ui import navigator, CFMENavigateStep, navigate_to
+from widgetastic.widget import Text, Checkbox
+from widgetastic_patternfly import BootstrapSelect, Button, Input
+from widgetastic_manageiq import ScriptBox
+
+from . import ServicesCatalogView
+
+
+class OrchestrationTemplatesAllView(ServicesCatalogView):
```

Comment:
Is this consistent in other modules? Consider 'OrchestrationTemplatesView' maybe?